databind/src/main/java/tech/ydb/yoj/databind/schema/GlobalIndex.java

```diff
@@ -30,4 +30,9 @@
      * List of annotated class fields representing index columns.
      */
     String[] fields();
+
+    /**
+     * Is unique index type
+     */
+    boolean unique() default false;
```

Comment:
Is "unique" an index type or a property of any index that can be created? If it's a dedicated index *type*, we should use an enum instead, I think.

Reply:
Changed to enum